modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="determining-upgrade-viability-conditiontype_{context}"]
+= Determining upgrade viability with conditionType
+
+The status of cluster operators is revealed with the value of conditionType, informing you of potential impacts to upgrading your cluster. 
```

Comment:
It is probably worth telling folks if we're talking about ClusterVersion, ClusterOperator, or both.  And maybe worth talking about user interfaces (`oc adm upgrade` and `oc get clusteroperators` on the command line, and `/k8s/cluster/config.openshift.io~v1~ClusterVersion/version` and `/settings/cluster/clusteroperators` in the web console)?  Or link to this section from existing docs discussing those UIs?  Or something...

Reply:
Great point - making sure I understand, you mean how/where folks can find this status in the first place? :)

Reply:
> It is probably worth telling folks if we're talking about ClusterVersion, ClusterOperator, or both

When we put a lot of conditions or push the reader to decide which content is applicable to which operator that's makes it less readable. So I am fine with the current text.

Reply:
> Or link to this section from existing docs discussing those UIs? Or something...

I agree that it might be worth pointing out how users can see these status condition.